unity-renderer/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -186,8 +192,26 @@ internal IEnumerator ProcessProfile(CullingControllerProfile profile)
                     Material mat = skr.sharedMaterial;
                     bool isAvatarRenderer = false;
 
+
                     if (mat != null && mat.shader != null)
-                        isAvatarRenderer = mat.shader.name == "DCL/Toon Shader";
+                    {
```

Comment:
Could you add a comment explaining what this `if` is supposed to do?

Reply:
Done!